docs/rfcs/xxx-pretranspile.md

```diff
@@ -0,0 +1,44 @@
+# Pre-transpiled Atom packages
+
+## Status
+
+Proposed
+
+## Summary
+
+This feature will enable package authors to use conventional npm tooling and package.json conventions to take advantage of JavaScript transpilers like Babel or TypeScript.
```

Comment:
![bitmoji](https://render.bitstrips.com/v2/cpanel/d436e0aa-e48e-4294-81a9-af3a39b8a276-f14a1e1d-64bc-41f2-8006-01b6c63adbd8-v1.png?transparent=1&palette=1&width=246)

Separate issue, but would we be able to use TypeScript in the GitHub package?  I don't know if you can incrementally use typescript or if it's an all or nothing kind of situation.

Reply:
> Separate issue, but would we be able to use TypeScript in the GitHub package?

Why yes. Yes we would. 😁 Although I think Relay will give us problems when we do.